react/src/components/DoubleTag.tsx

```diff
@@ -45,13 +45,24 @@ const DoubleTag: React.FC<{
             }}
             color={objValue.color}
           >
-            {!_.isUndefined(highlightKeyword) ? (
-              <TextHighlighter keyword={highlightKeyword}>
-                {objValue.label}
-              </TextHighlighter>
-            ) : (
-              objValue.label
-            )}
+            <div
+              // Since BAIText has own style for fontSize, I decided to not use BAIText here for ellipsis.
```

Comment:
You can use `inherit` for `fontSize`. It would be better to use BAIText to manage global changes later.